lib/ex_cell/adapters/cell_js.ex

```diff
@@ -49,7 +49,7 @@ defmodule ExCell.Adapters.CellJS do
   """
   def data_attribute(name, id, data \\ [], params \\ %{}), do:
     Keyword.merge(
-      data,
+      data || [],
```

Comment:
For future reference, this might be a bit confusing because it already defaults to `[]`. It would've been more readable if we added a specific pattern match for this case:
```ex
def data_attribute(name, id, nil, params \\ %{}), do:
  date_attribute(name, id, [], params)
```

Reply:
#40